terasoluna-gfw-functionaltest-web/src/main/resources/META-INF/spring/spring-mvc-common.xml

```diff
@@ -36,30 +36,26 @@
     <mvc:interceptor>
       <mvc:mapping path="/**" />
       <mvc:exclude-mapping path="/resources/**" />
-      <mvc:exclude-mapping path="/*/*.html" />
       <mvc:exclude-mapping path="/logging/traceLoggingInterceptor/customWarnHandling/**" />
       <bean class="org.terasoluna.gfw.web.logging.TraceLoggingInterceptor" />
     </mvc:interceptor>
     <mvc:interceptor>
       <mvc:mapping path="/**" />
       <mvc:exclude-mapping path="/resources/**" />
-      <mvc:exclude-mapping path="/*/*.html" />
       <mvc:exclude-mapping path="/transactiontoken/customTransactionStoreSize2/**" />
       <mvc:exclude-mapping path="/transactiontoken/customTransactionStoreSize1/**" />
       <bean class="org.terasoluna.gfw.web.token.transaction.TransactionTokenInterceptor" />
     </mvc:interceptor>
     <mvc:interceptor>
       <mvc:mapping path="/transactiontoken/customTransactionStoreSize2/**" />
       <mvc:exclude-mapping path="/resources/**" />
```

Comment:
Some `<mvc:exclude-mapping path="/resources/**" />` are not working. (not include in the mapping path)

Reply:
Removed not working <mvc:exclude-mapping path="/resources/**" />.
57b4165b46664a3116b91de2d56862a0ca4eb46b